examples/files.json

```diff
@@ -0,0 +1,44 @@
+{
+    "files": [
+        {
+            "_id": "a34e4793-c343-4930-973f-a45196289017",
+            "id": "a34e4793-c343-4930-973f-a45196289017",
+            "name": "dataset1_file1.fastq",
+            "format": "FASTQ",
+            "type": "Unaligned Reads",
+            "size": "432432423",
+            "md5sum": "9aa9aabaaf67287c332d85c0de825c03",
+            "category": "Whole Genome Sequencing"
+        },
+        {
+            "_id": "9e9d1ec1-aadc-4f45-a8e8-90d1e66f827f",
+            "id": "9e9d1ec1-aadc-4f45-a8e8-90d1e66f827f",
+            "name": "dataset1_file2.fastq",
+            "format": "FASTQ",
+            "type": "Unaligned Reads",
+            "size": "432475412",
+            "md5sum": "15d860fcd230fe88f6e09a954db70c2f",
+            "category": "Whole Genome Sequencing"
+        },
+        {
+            "_id": "e1645654-695e-4f52-8e5b-23debd73a678",
+            "id": "e1645654-695e-4f52-8e5b-23debd73a678",
+            "name": "dataset2_file1.bam",
+            "format": "BAM",
+            "type": "Aligned Reads",
+            "size": "324324324",
+            "md5sum": "459aaf513c2ccf5cc7fe6b2706226b0e",
```

Comment:
I would still prefer we used SHA-3 or SHA-2 instead of MD5 even if only for simple checksums.

Reply:
Thanks for pointing that out. I agree with your recommendation. This also highlights a point that the metadata field `md5sum` is restrictive. Instead we should say `checksum`